recipes/coz/all/test_package/conanfile.py

```diff
@@ -0,0 +1,19 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
+
+    def build(self):
+        cmake = CMake(self, build_type="RelWithDebInfo")
```

Comment:
```suggestion
        cmake = CMake(self)
```

Reply:
To work properly Coz tool requires debug information. That's why the build forced to `RelWithDebInfo` in the test_package.

> To run your program with coz, you will need to build it with debug information. You do not need to include debug symbols in the main executable: coz uses the same procedure as gdb to locate debug information for stripped binaries.

https://github.com/plasma-umass/coz

Reply:
Pretty please :pray:

```suggestion
        cmake = CMake(self, build_type="RelWithDebInfo") # To work properly Coz tool requires debug information https://github.com/plasma-umass/coz
```